backend/pom.xml

```diff
@@ -1,9 +1,9 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <parent>
-    <groupId>org.jboss</groupId>
-    <artifactId>dependency-analyzer</artifactId>
+    <groupId>org.jboss.da</groupId>
```

Comment:
If we put it under the ``org.jboss.pnc`` groupId then it would be at the same level as the submodules (e.g. SPI) within PNC. This could be regarded as a bit confusing?
I think having it under ``org.jboss.da`` is fine for now (and its not dissimilar to the layout that e.g. https://github.com/droolsjbpm/ uses).